.github/workflows/nightly_run.yml

```diff
@@ -323,16 +323,13 @@ jobs:
           - make_options_run_mode: $FAST_JIT_BUILD_OPTIONS
             make_options_feature: "-DWAMR_BUILD_MEMORY64=1"
     steps:
-      - name: checkout
-        uses: actions/checkout@v3
-
       - name: Install dependencies
-        uses: nick-fields/retry@v2
```

Comment:
the latest one seems to claim node 20 compat.
https://github.com/nick-fields/retry/releases/tag/v3.0.0

btw, is this retry necessary at all these days?

Reply:
I think it should be fine without it